src/hooks/useUser.js

```diff
@@ -1,10 +1,13 @@
 import { useMemo } from 'react'
 
-import { getShortNameFromClient } from 'lib/utils.js'
+import { getShortNameFromClient, getUserNameFromUrl } from 'lib/utils.js'
 
 function useUser({ userName: providedUserName, cozyClient }) {
   const userName = useMemo(
-    () => providedUserName || getShortNameFromClient(cozyClient),
+    () =>
+      providedUserName ||
+      getUserNameFromUrl() ||
```

Comment:
We're passing thins value down to prosemirror collab service by emitting a joined event. Are we sure that the consumer of this value are escaping the value?

Reply:
Yep, the mock of the collab service is also using names with a space